_src/static/sass/utilities/_variables.scss

```diff
@@ -193,34 +193,31 @@ $font-family-monospace: 'Bitstream Vera Sans Mono', Consolas, Courier, monospace
 // typography: config - scale
 $modular-scale-ratio: $perfect-fourth;
 
-$font-scale-small: 16;
-$font-scale-medium: 18;
+$font-scale-small: 14;
+$font-scale-medium: 16;
 $font-scale-large: 18;
 $font-scale-base: $font-scale-medium;
 
-// typography: sizes
+// typography: sizes (pixels)
```

Comment:
Please remove the "(pixels)" note from the comment here. I don't think its needed as we we are just defining a scale.